medical_surgical_appointment/models/medical_surgical_appointment.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class MedicalSurgicalAppointment(models.Model):
+
+    _name = 'medical.surgical.appointment'
+    _description = 'Medical Surgical Appointment'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
+
+    patient_id = fields.Many2one(
+        string='Patient',
+        comodel_name='medical.patient',
+        required=True,
+        track_visibility=True,
+        ondelete='restrict', index=True,
+        help='Patient Name',
+    )
+
+    state = fields.Selection(
+        [
+            ('draft', 'Draft'),
+            ('confirmed', 'Confirmed'),
+            ('arrived', 'Arrived'),
+        ],
+        required=True,
+        default='draft',
+        readonly=True,
+        track_visibility='onchange',
+    )
+
+    first_name = fields.Char(string='First Name', required=True)
+    last_name = fields.Char(string='Last Name')
+    second_last_name = fields.Char(string='Second Last Name')
+
+    vat = fields.Char(string='VAT', required=True)
+
+    # address = ... res.partner????
+
+    phone = fields.Char(string='Phone')
+    mobile = fields.Char(string='Mobile', required=True)
+    email = fields.Char(string='Email')
+
+    location_id = fields.Many2one(
+        string="Location",
+        comodel_name='res.partner',
+        domain=[('is_location', '=', True)],
+        track_visibility=True,
+        ondelete='restrict', index=True,
+    )  # FHIR Field: location
+
+    # center_id = fields -...
+
+    payor_id = fields.Many2one(
```

Comment:
should go to `medical_surgical_appointment_agreement`, also sub_payor, coverage_template, subscriber_id and authorization_number